mautic/mautic
# %contact% lang variable changed back to %lead%

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | N
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  /

## Description
Issue similar to https://github.com/mautic/mautic/pull/1643 but in singular. The issue is that the language strings containint `%contact%` didn't work and so this PR changes them back to `%lead%` to keep BC for language strings.

## Steps to reproduce the bug (if applicable)
Make sure you have a sent email action in your calendar. Look at the past months, there should be some. Hover over the action and the tooltip should contain the %contact% variable.

## Steps to test this PR
Apply this PR and check the email sent action again. The %contact% variable should be replaced with lead name.
